### PR TITLE
Testing part 2

### PR DESCRIPTION
**Pull request by Juank628**
## To-Do List: Update and clear tests
In this pull request we did the following:
- Created a test for the update method
- Created a test for the clear method
- Created a test for render after the update
- Created a test for render after clear

### CODE REVIEW

File: tests/editAndClear.test.js [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    expect(
+      collection.updateItem(2, 'description', 'test description'),
+    ).toStrictEqual([
+      {
+        description: 'first',
+        completed: false,
+        index: 1,
+      },
+      {
+        description: 'test description',
```
Comment (VICTOR-CHUKWU) [inline tests/editAndClear.test.js:34]: - [ ] excellent work, but you have to include a test for the editing task description

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @Juank628  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: tests/editAndClear.test.js (hunk 1 above)
Comment (Juank628) [reply tests/editAndClear.test.js:34]: Hi @VICTOR-CHUKWU, thanks for your comments. In fact, we are including the test for editing (update) the task description. This test starts in line 11 and ends in line 39. We are testing the description update from "second" to "test description". Could you check again, please?

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Juank628 

Your project is complete!  🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Test ok
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**